Makefile

```diff
@@ -94,6 +94,7 @@ initialize_db:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
+	rm -rf ./user-media/ ./tmp/
```

Comment:
Since those folders don't exist when cloning the repository, I don't think any code relies on them. however, I can delete their contents only.

I hear you about potential re-usage, but this command should not be called without a db reset. Otherwise, we'd just be in the same situation we are in now (lost sync between db and filesystem). I looked through the file and couldn't find any other use cases at this point. Maybe lets refactor once we have that?